image_class_info:
  image_folder_name: '/home/dy/test_image/'
  object_classes:
    - juice
    - vitamin_drink
    - tea
    - fonta
    - yogurt
    - cola
    - red_pretz
    - fuck
  image_num: 10
